apache-mesos-mixin/dashboards/apache-mesos-overview.libsonnet

```diff
@@ -0,0 +1,1387 @@
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [
+        {
+          options: {
+            '0': {
+              color: 'red',
+              index: 1,
+              text: 'Not OK',
+            },
+            '1': {
+              color: 'green',
+              index: 0,
+              text: 'OK',
+            },
+          },
+          type: 'value',
+        },
+        {
+          options: {
+            match: 'null',
+            result: {
+              color: 'text',
+              index: 2,
+              text: '-',
+            },
+          },
+          type: 'special',
+        },
+      ],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: '',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    text: {},
+    textMode: 'auto',
+  },
+  pluginVersion: '9.5.2-cloud.2.0cb5a501',
+};
+
+local allocatorRow = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      '',
+      datasource=promDatasource,
+      legendFormat='',
+    ),
+  ],
+  type: 'row',
+  title: 'Allocator',
+  collapsed: false,
+};
+
+local allocationRunsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'max by(mesos_cluster) (rate(mesos_master_allocation_run_ms_count{job=~"$job", instance=~"$instance"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{mesos_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Allocation runs',
+  description: 'The rate of how often the allocator is performing allocations.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'allocations/sec',
```

Comment:
Can you reduce this to either:
- `allocs/s`
- `allocations/s`

Having `/sec` is quite wordy and actually ends up having issues with staying within the bounding box as seen here:
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/38e4eb42-2591-4655-a76c-747a9ab5de82)